airflow/providers/slack/operators/slack.py

```diff
@@ -48,7 +52,7 @@ def __init__(
         self,
         *,
         slack_conn_id: str = SlackHook.default_conn_name,
-        method: str | None = None,
+        method: str,
```

Comment:
method can't be `None` because `SlackHook.call` method expected `str` however this previously doesn't failed on static checks, my assumption due to the wrong signature of `SlackAPIOperator.execute` method.  After I change it to correct one it starts failed with an error

```console
airflow/providers/slack/operators/slack.py:100: error: Argument 1 to "call" of
"SlackHook" has incompatible type "Optional[str]"; expected "str"  [arg-type]
            self.hook.call(self.method, json=self.api_params
```